### DISCUSSION TITLE
How can I increase the width of gr.textbox type settings?

### DISCUSSION THREAD

**Discussion by cjps-br** [vladmandic/sdnext#2171] [Q&A] [answered]
I have an extension that I created for personal use and that uses system\settings to store some settings. I noticed that the gr.textbox type settings are "narrow" and cannot show the entire stored value. How could I increase the width of this type of settings? Attached is an example of how the gr.textbox type is being shown

![Captura de tela 2023-09-11 120234](https://github.com/vladmandic/automatic/assets/75840971/b2b2e12d-b836-4b5b-9db2-5351a2d87d30)

**Comment by cjps-br** [comment]
Well, I solved the issue by adding characters to the item shared.opts.add_option as follows: shared.opts.add_option("cjp_dir_destino_na_copia _______________________________________________________________________.",

I don't believe it's the best solution, but I don't want to mess with the default css. The result was like the attached image.
![Captura de tela 2023-09-12 124405](https://github.com/vladmandic/automatic/assets/75840971/1e444921-8c86-46bb-afa2-b87cd6d938f0)

**Reply by vladmandic** [reply, marked as answer]
i'm going to revert the change that reduced the size of controls as there were too many sideeffects.
